content/docs/guides/hedgedoc.md

```diff
@@ -1,147 +1,113 @@
 
 # HedgeDoc
 
-Use Pomerium to secure access to HedgeDoc by providing identity and context.
+In this guide, you'll learn how to run a HedgeDoc web application behind Pomerium. You'll also add an authorization policy that enables Pomerium to gran or deny access based on policy criteria.
 
 ## What is HedgeDoc?
 
-[HedgeDoc](https://hedgedoc.org/) is a collaborative, web-based Markdown editor that allows you to create notes, graphs, and diagrams in your browser.
+[HedgeDoc](https://hedgedoc.org/) is a collaborative, web-based Markdown editor that allows you to create notes, graphs, and diagrams in your browser. You can share a link to your documents with other members of your organization so you can collaborate in real time.
 
-You can share a link to your document with other members of your organization so you can collaborate in real time.
+## How to secure HedgeDoc with Pomerium
 
-## Integrate Pomerium with HedgeDoc
+HedgeDoc is an open-source project that you can serve behind a [reverse proxy](https://docs.hedgedoc.org/guides/reverse-proxy/).
 
-HedgeDoc is an open-source, self-hosted software that allows you to configure authentication and authorization with a [reverse proxy](https://docs.hedgedoc.org/guides/reverse-proxy/).
+In this guide, you'll configure Pomerium to authenticate and authorize users. To do this, you'll add an authorization policy that grants or denies access based on the policy criteria.
 
-You can configure Pomerium so that it authenticates and authorizes users against an Identity Provider (IdP) before users can access HedgeDoc's services.
-
-The flow looks like this:
-
-1. User requests to access the secured app
-2. Pomerium identifies user
-3. Pomerium checks if user is authorized to access routes defined in the authorization [`policy`](https://www.pomerium.com/docs/reference/routes/policy)
-4. If authorized, Pomerium authenticates user against IdP
-5. After IdP establishes user's identity, Pomerium checks permissions against the secured app
-6. User can make authorized requests in the secured app
-
-![Pomerium IAP flow](./img/hedgedoc/pom-auth-flow.png)
-
-### Set up your environment
-
-This guide uses Docker containers to create the minimal development environment required to run Pomerium with another service. Visit the [Pomerium using Docker](https://www.pomerium.com/docs/quickstart) quickstart for more information.
+### Before you start
 
 To complete this guide, you need:
 
 - [Docker](https://www.docker.com/)
 - [Docker Compose](https://docs.docker.com/compose/install/)
-- A running Pomerium instance
-- A [pre-configured IdP](https://www.pomerium.com/docs/identity-providers)
-
-:::tip
+- [mkcert](https://github.com/FiloSottile/mkcert#installation)
 
-**Note**
+:::note
 
-This guide uses [GitHub](https://www.pomerium.com/docs/identity-providers/github) as the pre-configured IdP.
+If you completed our [**Quickstart guide**](/docs/quickstart), you should have a working Pomerium project with the following YAML files:
```

Comment:
Feels like I'm missing something here. 
"...the following YAML files:" with a colon but then there's nothing after it. Please check.

Reply:
The YAML files are listed below as bullet points:
![SCR-20240227-lqlj](https://github.com/pomerium/documentation/assets/60552605/6d99325f-6665-4c49-82ba-d048f72688af)